tests/test_libzim.py

```diff
@@ -274,3 +275,58 @@ def get_redirect_url(self):
     with File(path) as reader:
         assert reader.get_article(redirect_url).is_redirect
         assert reader.get_article(redirect_url).get_redirect_article().longurl == url
+
+
+@pytest.mark.parametrize(
+    "no_method", [m for m in dir(Article) if m.split("_", 1)[0] in ("get", "is", "should")],
+)
```

Comment:
I didn't know that we could directly parametrize a test. I was using to parametrize a fixture and use the fixture in the test. Good to know :)